isaric/parsers/ccp-uganda.toml

```diff
@@ -0,0 +1,895 @@
+is_present = { field = "chest_pain", ref = "Y/N/NK" }
+
+[[observation]]
+name = "chest_pain"
+date = { field = "vitals_date" }
+phase = "study"
+is_present = { field = "chest", ref = "Y/N/NK" }
+
+[[observation]]
+name = "conjunctivitis"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "conjunc", ref = "Y/N/NK" }
+
+[[observation]]
+name = "conjunctivitis"
+date = { field = "vitals_date" }
+phase = "study"
+is_present = { field = "conjunct", ref = "Y/N/NK" }
+
+[[observation]]
+  name = "cough"
+  phase = "admission"
+
+  [observation.date]
+    combinedType = "firstNonNull"
+    fields = [
+      { field = "signs_symps_date" },
+      { field = "vitals_date" },
+      { field = "investigate_date" }
+    ]
+
+  [observation.is_present]
+    combinedType = "firstNonNull"
+    fields = [
+      { field = "cough", ref = "Y/N/NK" },
+      { field = "dry_cough", ref = "Y/N/NK" },
+      { field = "cof", ref = "Y/N/NK" },
+      { field = "sick_symp", values = { 1 = false, 2 = true, 3 = false, 4 = false } },
+      { field = "wet_cough", ref = "Y/N/NK" },
+      { field = "blood_cough", ref = "Y/N/NK" },
+    ]
+
+  [observation.if]
+    any = [
+      { cough = { "!=" = "" } },
+      { cof = {"!=" = "" } },
+      { sick_symp = {"!=" = "" } },
+      { dry_cough = { "!=" = "" } },
+      { wet_cough = { "!=" = "" } },
+      { blood_cough = { "!=" = "" } },
+    ]
+
+[[observation]]
+name = "cough_with_haemoptysis"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "blood_cough", ref = "Y/N/NK" }
+
+[[observation]]
+name = "cough_with_sputum_production"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "wet_cough", ref = "Y/N/NK" }
+
+[[observation]]
+name = "diarrhoea"
+date = { field = "investigate_date" }
+phase = "admission"
+is_present = { field = "sick_symp", values = { 1 = false, 2 = false, 3 = true, 4 = false } }
+
+[[observation]]
+name = "diarrhoea"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "diarrhea", ref = "Y/N/NK" }
+
+[[observation]]
+name = "diarrhoea"
+date = { field = "vitals_date" }
+phase = "study"
+is_present = { field = "diarr", ref = "Y/N/NK" }
+
+[[observation]]
+name = "ear_pain"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "ear_pain_v2", ref = "Y/N/NK" }
+
+[[observation]]
+name = "ear_pain"
+date = { field = "vitals_date" }
+phase = "study"
+is_present = { field = "ear_pain", ref = "Y/N/NK" }
+
+[[observation]]
+name = "fatigue_malaise"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "fatigue", ref = "Y/N/NK" }
+
+[[observation]]
+name = "fatigue_malaise"
+date = { field = "vitals_date" }
+phase = "study"
+is_present = { field = "fatig", ref = "Y/N/NK" }
+
+[[observation]]
+name = "headache"
+date = { field = "signs_symps_date" }
+phase = "admission"
+is_present = { field = "headache", ref = "Y/N/NK" }
```

Comment:
phase = study? - there's a few of these, the header of that section of the form suggests this is a daily form '_during_ admission' suggests during period in hospital as it's labelled to repeat as many days as required.

Reply:
thanks. 
I find this can get blurry, not knowing when some observations were made, at the time of admission or during admission / within the study ...